### PR TITLE
Add conditional CUDA support with env variable

### PR DESCRIPTION
**Pull request by richard-rogers**
Pass device argument to enable GPU

### CODE REVIEW

**Comment by richard-rogers** [comment]
I will add a `WHYLABS_DISABLE_GPU` env variable, and see if I can enable GPU for sentiment.

File: langkit/toxicity.py [hunk 1]
```diff
@@ -4,6 +4,9 @@
 from whylogs.experimental.core.udf_schema import register_dataset_udf
 from langkit import LangKitConfig, lang_config, prompt_column, response_column
 
+import torch
+
+_device = 0 if torch.cuda.is_available() else -1
```
Comment (richard-rogers) [inline langkit/toxicity.py:9]:
```suggestion
import os
import torch

_USE_CUDA = torch.cuda.is_available() and not bool(os.environ.get("WHYLABS_DISABLE_CUDA", False))
_device = 0 if  _USE_CUDA else -1
```

File: langkit/transformer.py [hunk 2]
```diff
@@ -3,6 +3,11 @@
 from torch import Tensor
 import numpy as np
 
+import torch
+
+_device = "cuda" if torch.cuda.is_available() else "cpu"
```
Comment (richard-rogers) [inline langkit/transformer.py:8]:
```suggestion
import os
import torch

_USE_CUDA = torch.cuda.is_available() and not bool(os.environ.get("WHYLABS_DISABLE_CUDA", False))
_device = "cuda" if _USE_CUDA else "cpu"
```

File: langkit/toxicity.py [hunk 3]
```diff
@@ -4,6 +4,11 @@
 from whylogs.experimental.core.udf_schema import register_dataset_udf
 from langkit import LangKitConfig, lang_config, prompt_column, response_column
 
+import os
+import torch
+
+_USE_CUDA = torch.cuda.is_available() and not bool(os.environ.get("WHYLABS_DISABLE_CUDA", False))
```
Comment (jamie256) [inline langkit/toxicity.py:10]: It’s not disabling CUDA, suggest rename the env var to indicate it’s disabling running LangKit on CUDA?

**Review by jamie256 — APPROVED** [review]
LGTM!

**Pull request merged**